crypto/src/sha3/mod.rs

```diff
@@ -1,35 +1,513 @@
-pub use sha3::*;
```

Comment:
Please follow the blake2s path - keep two implementations under features, one via "standard" implementation (sha3::Keccak256) and one via your code below

Reply:
and the feature i should keep out of default ? just optional ?

Reply:
i put in default btw